lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1717,6 +1721,24 @@ protected function _renderTemplates($content) {
 			$rendered[$type] = implode("\n", $rendered[$type]);
 			$rendered[$type] = rtrim($rendered[$type], "\n");
 		}
+
+		/* Embed images inline in html templates */
+		if (!empty($rendered['html'])) {
+			$rendered['html'] = str_replace(array('file:', 'file://', 'cid://'), 'cid:', $rendered['html']);
+			if (preg_match_all('~(["\'])cid:([^\1]+)\1~iU', $rendered['html'], $img)) {
```

Comment:
Won't this also match `"cid:.*"` in text/outside of an img tag? Looks like this could create malformed message content.